pyiron_workflow/executors/wrapped_executorlib.py

```diff
@@ -0,0 +1,104 @@
+import inspect
+from collections.abc import Callable
+from typing import ClassVar
+
+from executorlib import BaseExecutor, SingleNodeExecutor, SlurmClusterExecutor
+from executorlib.task_scheduler.file.subprocess_spawner import execute_in_subprocess
+from executorlib.task_scheduler.file.task_scheduler import FileTaskScheduler
```

Comment:
@liamhuber I would like to prevent deep links into the executorlib package, as I only promise the `executorlib.X` and `executorlib.api.X` to remain stable. Can you tell me why you need the `LocalFileExecutor`? I currently do not see any functionality which is not covered by the `SingleNodeExecutor` in a more efficient way. I primarily intended the `LocalFileExecutor` for debugging purposes.

Reply:
I am also including it for debugging purposes, I don't intend to include it in the api here. If I can get the GitHub CI working with Slurm later I'll probably remove it, but so far every time I've encountered a bug/problem with the slurm executor over on the cluster I've been able to reproduce it with the local file executor, solve that locally, and the solution works for slurm too. So it's super useful to me at the moment but never intended for users.

Reply:
Yes, that is exactly the use case it is designed for, maybe I should include it in `executorlib.api` as it is helpful for developers to test their code while it is might be confusing for users. In addition it might also make sense to adjust the name, I thought about `TestFileExecutor` or `LocalClusterExecutor`. The first as a reference that it should only be used for testing while the second is more focused on highlighting the similarity to the other cluster executors.

Reply:
I opened a corresponding pull request in the executorlib repository https://github.com/pyiron/executorlib/pull/714